src/docs/bibtex/all.bib

```diff
@@ -362,6 +362,17 @@ @article{BleiLafferty:2007
   pages = {17--37}
 }
 
+@article{Blurton2012,
+	author = {Blurton, Steven P. and Kesselmeier, Miriam and Gondan, Matthias},
+	journal = {Journal of Mathematical Psychology},
+	number = {6},
+	pages = {470-475},
+	title = {Fast and accurate calculations for cumulative first-passage time distributions in Wiener diffusion models},
```

Comment:
This needs to be {Wiener} to get it capitalized in the output and the pages need to be -- rather than whatever is there.  You need to check the output.